packages/server/src/errors.ts

```diff
@@ -327,11 +259,8 @@ export function formatApolloErrors(
         return enrichError(err as Partial<GraphQLError>, debug);
       } else {
         // obscure error
-        const newError = fromGraphQLError(
-          new GraphQLError('Internal server error'),
-        );
-        return enrichError(newError, debug);
```

Comment:
Is removing this a behavior change?

Reply:
BTW. `debug` was always false because of the above if.

Reply:
Add modified test to ensure `code` is present.